### DISCUSSION TITLE
Find and or set working code folder for PC-BASIC in Win 11

### DISCUSSION THREAD

**Discussion by Raspberry45man** [robhagemans/pcbasic#181] [Q&A]
After reading the documentation PDF, it is not clear how to find the default working drive/directory/folder for any BAS files created.  Also, I would like to change that location to my preferred working code drive/directory/folder on a USB drive.  If possible to do this, can I also EDIT the INI file to reflect the location of my preferred drive/directory/folder, looking for correct syntax to do this.

**Comment by robhagemans** [comment]
I've spent an inordinate amount of time, literally hours, trying to
formulate the documentation carefully and precisely so as to explain to
Windows users where to find this directory. So can you please be more
specific as to what you read and how it is unclear?
